addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -3383,7 +3383,15 @@ export class OdooEditor extends EventTarget {
                         const textFragments = splitAroundUrl[i].split(/\r?\n/);
                         let textIndex = 1;
                         for (const textFragment of textFragments) {
-                            this._applyCommand('insertText', textFragment);
+                            const modifiedTextFragment = textFragment.replace(/( {2,})/g, match => {
```

Comment:
Thanks for the fix! Please add a comment that says: "// Replace consecutive spaces by alternating nbsp."